### PR TITLE
Window resize

### PR DESCRIPTION
**Pull request by Maximilian-Reuter**

### CODE REVIEW

File: jars/shaderpacks/default/shaders/gui.vert [hunk 1]
```diff
@@ -4,11 +4,11 @@ layout(location = 0) in vec3 position_in;
 layout(location = 1) in vec2 uv_in;
 layout(location = 2) in vec3 color_in;
 
-layout(shared, std140) uniform gui_uniforms {
-    float viewWidth;
-    float viewHeight;
-    mat4 gbufferModelView;
+layout(binding = 0,std140) uniform gui_uniforms {
```
Comment (DethRaid) [inline jars/shaderpacks/default/shaders/gui.vert:7]: We shouldn't need to specify the binding here, Nova should pick it up at runtime. Plus, if you do specify the binding, you'll have to put that on the UBO every time it's used in a shader

File: src/main/cpp/render/objects/shaders/uniform_buffers_definitions.cpp [hunk 2]
```diff
@@ -10,15 +10,15 @@
 
 namespace nova {
     el::base::Writer &operator<<(el::base::Writer &out, gui_uniforms uniforms) {
-        out << "gbufferModelView: " << uniforms.gbufferModelView
-            << " gbufferProjection:" << uniforms.gbufferProjection
+        out << "gbufferModelView: \n" << uniforms.gbufferModelView
```
Comment (DethRaid) [inline src/main/cpp/render/objects/shaders/uniform_buffers_definitions.cpp:13]: I probably should have done this. All well.

File: src/main/cpp/render/uniform_buffer_store.cpp [hunk 3]
```diff
@@ -69,7 +75,7 @@ namespace nova {
 
     void uniform_buffer_store::update_per_frame_uniforms() {
         // TODO: Fill in the per frame uniforms
-        per_frame_uniforms_buffer.send_data(per_frame_uniform_variables);
+        //per_frame_uniforms_buffer.send_data(per_frame_uniform_variables);
```
Comment (DethRaid) [inline src/main/cpp/render/uniform_buffer_store.cpp:78]: Why is this commented out

**Review by DethRaid — CHANGES_REQUESTED** [review]
Fix the logging and I'll merge

File: src/main/cpp/render/uniform_buffer_store.cpp (hunk 3 above)
Comment (Maximilian-Reuter) [reply src/main/cpp/render/uniform_buffer_store.cpp:78]: the per frame uniforms don't get any data set and they do produce an Invalid operation or Value down the line somewhere. until we actually need it, it just complicates things as opengl stuff has a tendency to explode in your face

**Review by DethRaid — APPROVED** [review]

**Pull request merged**